.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I haven't committed any vscode config yet as @Araneidae doesn't use vscode. However as there are now two of us doing this, maybe we should. What do you think @Araneidae ?

Reply:
Thinking about it, if we do go down that route we should probably just bring across all the .vscode settings from the template module, rather than what I've done here and just a small selection to get one particular piece working as I wanted.